std_msgs/Header header
Detection[] data
uint8 camera_index
